core/store/orm/orm.go

```diff
@@ -172,6 +172,19 @@ func (orm *ORM) FindJob(id *models.ID) (models.JobSpec, error) {
 	return job, orm.preloadJobs().First(&job, "id = ?", id).Error
 }
 
+// FindJobWithErrors looks up a Job by its ID.
+func (orm *ORM) FindJobWithErrors(id *models.ID) (models.JobSpec, error) {
+	orm.MustEnsureAdvisoryLock()
```

Comment:
Why the avisory lock?

Reply:
This was resolved without an answer

Reply:
whoops sorry - thought I had gotten rid of that lock